<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DemoFirstPart</class>
 <widget class="QWidget" name="DemoFirstPart">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1102</width>
    <height>843</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="rgbLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>640</width>
     <height>360</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Color Image</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="depthLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>470</y>
     <width>640</width>
     <height>360</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Depth Image</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="capture">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>780</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>CAPTURE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startStreaming">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>780</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>START</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea_2">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>470</y>
     <width>411</width>
     <height>301</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>409</width>
      <height>299</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="8" column="0">
      <widget class="QSlider" name="DepthUnit">
       <property name="minimum">
        <number>10</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="singleStep">
        <number>10</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QSlider" name="Exposure_">
       <property name="minimum">
        <number>20</number>
       </property>
       <property name="maximum">
        <number>166000</number>
       </property>
       <property name="singleStep">
        <number>10</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QSlider" name="Gain_2">
       <property name="minimum">
        <number>16</number>
       </property>
       <property name="maximum">
        <number>248</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QSlider" name="LaserPower">
       <property name="maximum">
        <number>360</number>
       </property>
       <property name="singleStep">
        <number>10</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QCheckBox" name="AutoExposure_2">
       <property name="text">
        <string>Auto Exposure</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>Gain : 16.000 ~ 248.000  </string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>Laser Power : 0.000 ~ 360.000  </string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Exposure 20.000 ~ 166000.000  </string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>Depth Unit : 0.0001 ~ 0.01</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QScrollArea" name="scrollArea_3">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>120</y>
     <width>411</width>
     <height>291</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>392</width>
      <height>470</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="4" column="0">
      <widget class="QCheckBox" name="AutoExposure">
       <property name="text">
        <string>Auto Exposure</string>
       </property>
      </widget>
     </item>
     <item row="17" column="0">
      <widget class="QCheckBox" name="AutoWhiteBalance">
       <property name="text">
        <string>Enable Auto White Balance</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QSlider" name="Exposure">
       <property name="minimum">
        <number>-8</number>
       </property>
       <property name="maximum">
        <number>0</number>
       </property>
       <property name="value">
        <number>-2</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Exposure : -8.000 ~ 0.000 </string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QSlider" name="Gain">
       <property name="maximum">
        <number>128</number>
       </property>
       <property name="value">
        <number>35</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Gamma : 100.000 ~ 500.000  </string>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QSlider" name="Gamma">
       <property name="minimum">
        <number>100</number>
       </property>
       <property name="maximum">
        <number>500</number>
       </property>
       <property name="value">
        <number>234</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <widget class="QSlider" name="Hue">
       <property name="minimum">
        <number>-180</number>
       </property>
       <property name="maximum">
        <number>180</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="15" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Sharpness : 0.000 ~ 100.000  </string>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Hue : -180.000 ~ 180.000  </string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Gain : 0.000 ~ 128.000  </string>
       </property>
      </widget>
     </item>
     <item row="13" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Saturation : 0.000 ~ 100.000   </string>
       </property>
      </widget>
     </item>
     <item row="14" column="0">
      <widget class="QSlider" name="Saturation">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>22</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Brightness : -64.000 ~ 64.000</string>
       </property>
      </widget>
     </item>
     <item row="16" column="0">
      <widget class="QSlider" name="Sharpness">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QSlider" name="Brightness">
       <property name="minimum">
        <number>-64</number>
       </property>
       <property name="maximum">
        <number>64</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Contrast : 0.000 ~ 100.000 </string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QSlider" name="Contrast">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="19" column="0">
      <widget class="QSlider" name="WhiteBalance">
       <property name="minimum">
        <number>2800</number>
       </property>
       <property name="maximum">
        <number>6500</number>
       </property>
       <property name="singleStep">
        <number>10</number>
       </property>
       <property name="pageStep">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="18" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>White Balance : 2800.000 ~ 6500.000 </string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLabel" name="camName">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>50</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="deviceFirmware">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>90</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>카메라 펌웨어 : </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="deviceFirmware_2">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>70</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>시리얼 넘버    : </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="deviceName">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>50</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>연결된 장치    : </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="serialNum">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>70</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="firmwareVer">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>90</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>rgbLabel</zorder>
  <zorder>label_2</zorder>
  <zorder>depthLabel</zorder>
  <zorder>label_4</zorder>
  <zorder>capture</zorder>
  <zorder>startStreaming</zorder>
  <zorder>scrollArea_3</zorder>
  <zorder>camName</zorder>
  <zorder>deviceFirmware</zorder>
  <zorder>deviceFirmware_2</zorder>
  <zorder>deviceName</zorder>
  <zorder>serialNum</zorder>
  <zorder>firmwareVer</zorder>
  <zorder>scrollArea_2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
